en/asgardeo/docs/quick-starts/mcp-auth-server.md

```diff
@@ -5,20 +5,19 @@ template: templates/quick-start.html
 <script>
   const meta = {
     what_you_will_learn: [
-      "Create new Node.js MCP server",
-      "Install <a href='https://www.npmjs.com/package/mcp-auth' target='_blank' rel='noopener noreferrer'>MCP Auth</a> package",
+      "Create a new MCP server in Typescript",
+      "Install <a href='https://www.npmjs.com/package/@asgardeo/mcp-express' target='_blank' rel='noopener noreferrer'>Asgardeo MCP SDK</a>",
       "Set up {{ product_name }} for user authentication",
-      "Define an MCP tool to return profile information"
+      "Access MCP tools and resources securely"
     ],
     prerequisites: [
       "About 15 minutes",
       "<a href='{{ base_path }}/get-started/create-asgardeo-account/'>{{ product_name }} account</a>",
       "Install Node.js on your system",
       "Make sure you have a JavaScript package manager like npm, yarn, or pnpm",
-      "Install <a href='https://claude.ai/download' target='_blank' rel='noopener noreferrer'>Claude Desktop</a>",
       "A favorite text editor or IDE"
     ],
-    source_code: "<a href='https://github.com/sagara-gunathunga/mcp-auth-qsg' target='_blank' class='github-icon'>MCP-Auth Sample</a>"
+    source_code: "<a href='https://github.com/ayshsandu/mcp-auth-quickstart' target='_blank' class='github-icon'>MCP-Auth Sample</a>"
```

Comment:
It seems the server.ts files contains a couple of additional line of code (dotenv) and that was not mention in this guide. @ayshsandu shall we make sure to sync both the sample and the guide?